elixir/apps/domain/lib/domain/policies/policy/query.ex

```diff
@@ -125,6 +125,24 @@ defmodule Domain.Policies.Policy.Query do
         type: {:string, :uuid},
         fun: &filter_by_actor_group_id/2
       },
+      %Domain.Repo.Filter{
```

Comment:
Is there a reason why we added this filter? It's not used anywhere.

Reply:
Well, I started with individual filters at first and then realized I needed a combined filter.  I figured these filters may be useful at some point so left them in, but happy to remove them if you'd like.

Reply:
@bmanifold let's remove them as they are half of the function that is used, but we won't need to ignore them everywhere

Reply:
👍 Sounds good, I'll get this updated.

Reply:
Updated.